vector/src/main/java/im/vector/app/features/home/room/detail/composer/TextComposerViewModel.kt

```diff
@@ -152,7 +152,11 @@ class TextComposerViewModel @AssistedInject constructor(
         withState { state ->
             when (state.sendMode) {
                 is SendMode.REGULAR -> {
-                    when (val slashCommandResult = CommandParser.parseSplashCommand(action.text)) {
+                    val slashCommandResult =
+                            CommandParser.parseSplashCommand(action.text)
+                                    .takeUnless { it.command?.isAvailable(vectorPreferences.developerMode()) == false }
```

Comment:
I prefer use your previous code "val parsedCommand = Command.values().firstOrNull { it.command == slashCommand }" to convert typed command into the enum value